installer/charts/rhtap-openshift/templates/projects.yaml

```diff
@@ -6,16 +6,21 @@
 {{- end -}}
 
 {{- range $p := $projects }}
-  {{- if and
-        (not (hasPrefix "openshift-" $p))
-        (not (lookup "project.openshift.io/v1" "Project" "" $p ))
-  }}
+  {{- if not (lookup "project.openshift.io/v1" "Project" "" $p ) }}
+    {{- if hasPrefix "openshift-" $p }}
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ $p }}
+    {{- else }}
```

Comment:
Need this cause project request will reject an namespace that starts with openshift-

Reply:
Something is causing the openshift-sorage to be recreated when helm upgrade happens. Lets hold on merging this PR for now, unit I can verify why.

Reply:
I know what the problem is, fixing

Reply:
This is fixed